etc/cvmfs/config.d/ligo.osgstorage.org.conf

```diff
@@ -1,5 +1,5 @@
 CVMFS_EXTERNAL_URL="https://stashcache.t2.ucsd.edu:8444//user/ligo/;https://osg-kansas-city-stashcache.nrp.internet2.edu:8444//user/ligo/;https://osg-chicago-stashcache.nrp.internet2.edu:8444//user/ligo/;https://osg-new-york-stashcache.nrp.internet2.edu:8444//user/ligo/;https://dtn2-daejeon.kreonet.net:8444//user/ligo/"
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://fiona.uvalight.net:8444//user/ligo/;https://stashcache.gravity.cf.ac.uk:8444//user/ligo/;https://xcache.cr.cnaf.infn.it:8443//user/ligo/"
+CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://fiona.uvalight.net:8444//user/ligo/;https://stashcache.gravity.cf.ac.uk:8444//user/ligo/;https://xcache.cr.cnaf.infn.it:8443//user/ligo/;https://xcachevirgo.pic.es:8444//user/ligo/"
```

Comment:
Are you sure that 8444 is the right port?  If I try to connect to it, it says Connection refused.

Reply:
@DrDaveD you were right they changed it without telling me it is 8443. I fixed it.